product.json

```diff
@@ -233,7 +232,6 @@
 		{
 			"name": "posit.publisher",
 			"version": "1.10.0",
```

Comment:
Can we bump the Publisher extension to their latest release (1.12.0), while we have this PR open? It has some improved behavior for us around their custom panel:

https://github.com/posit-dev/publisher/blob/main/CHANGELOG.md

Reply:
Certainly let's switch this back if it is a blocker to getting this PR merged. I apologize that I didn't notice it wasn't on Open VSX before!

Reply:
I was able to upgrade this to 1.12.1 and the signing on my build branch works as expected: https://github.com/posit-dev/positron-builds/actions/runs/14733437765 🎉